docs/partners/certification.md

```diff
@@ -0,0 +1,58 @@
+---
+id: certification 
+title: Certification 
+---
+
+Certification is the process by which Cortex XSOAR validates and marks the quality of the Content Pack, free or paid. It correlates with the confidence a customer should have in the delivered content offering. Certified Packs indicate the highest level of quality and reliability in our ecosystem. Palo Alto Networks sets detailed standards for certification but see below for high level guidance.
+
+
+><i>Note:</i> certification is separate from support; all Partner-submitted Packs must be supported by the Partner, regardless if the Pack is certifed or not. 
+
+
+## Levels of Certification
+
+When beginning the development process, the Contributor specifies the certification level they intend to achieve, and this will dictate the tasks the Contributor must complete. 
+
+- Not Certified: The Pack has gone through full security testing and basic functionality testing by Palo Alto Networks.
+
+- Certified: The Pack has gone through full security testing and full functionality testing including full coding standards, best practices check, documentation review and a demonstration. Recertification requirements to ensure retention of quality shall apply (see below).
+
+It is up to Palo Alto Networks at its discretion to mark a piece of content with a specific certification.
+
+
+## Guidance
+
+All Paid packs <b>must</b> be certified. And if a paid Pack has dependencies on another Pack from the same Partner, that other Pack must also be certified. Since those dependant Packs work together, the whole "solution" must be certified. 
+
+We encourage our Partners to achieve certification as this will up-level the quality of content in our ecosystem. 
+
+
+## Requirements
+
+While we encourage certified Packs, Certification requires several additional tasks and thus may not be applicable to all Partners:
+
+1. Forward compatibility testing — partners must ensure full Pack compatibility with future versions of Cortex XSOAR via a self-certify process. This process shall occur with all major and minor release of Cortex XSOAR (3-4/yr)
+
+1. Documentation — full documentation for the entire pack reviewed by Palo Alto Networks. 
+
+1. Mature use cases and content — including full coverage of Partner’s APIs and features; playbooks to allow customers to demonstrate the value, layouts to highlight fields, etc. 
+
+1. Coding best practices — adherence to best practices for coding standards and a deeper inspection of code. 
+
```

Comment:
What about mentioning the need in a test PB that will be shipped together with a working testing instance?

Reply:
I would call it `Full Functionality Testing`

Reply:
It should contain testing the functionality of the pack by running the commands, playbooks by us. It requires from partner provide us an instance.
It also requires create a test playbook that tests for pack functionality and this test playbook will run periodically and perform regular regression testing.

Reply:
Changed #1 to Full Functionality Testing
Added new bullet for prod access.